config/vufind/markdown.ini

```diff
@@ -12,6 +12,30 @@
 ;renderer[inner_separator] = "\n"
 ; String to use for rendering soft breaks
 ;renderer[soft_break] = "\n"
+; The maximum nesting level for blocks (default: infinite). Setting this to a
+; positive integer can help protect against long parse times and/or segfaults if
+; blocks are too deeply-nested.
+;max_nesting_level = 10
+
+; Which extension you want to activate. List of extension names separated by comma.
+; Available extensions as of league/commonmark version 2.2:
+;   Attributes, Autolink, DefaultAttributes, DescriptionList, DisallowedRawHtml,
+;   ExternalLink, Footnote, FrontMatter, HeadingPermalink, Mention, SmartPunct,
+;   Strikethrough, Table, TableOfContents, TaskList.
+; More about extensions: https://commonmark.thephpleague.com/2.2/extensions/overview/
+; Some of them could have a configuration, see sections below.
+; Note: FrontMatter does enable to inject the title other then main heading. Just add
+; something like this at the very beginning of your markdown file:
+;---
+;title: "Some specific title"
+;---
+; See more about FrontMatter extension here:
+; https://commonmark.thephpleague.com/2.2/extensions/front-matter/
+
+extensions = Autolink,DisallowedRawHtml,Strikethrough,Table,TaskList
+
+; CommonMarkCore extension is always enabled by default
+[CommonMarkCore]
```

Comment:
Adding this section effectively move some settings out of [Markdown] and into [CommonMarkCore]. Do we need to worry about backward compatibility? (I imagine the chances of people being affected by this change are relatively low... but maybe if nothing else a changelog note is justified).

Reply:
That is handled in `MarkdownFactory::getConfigForCoreExtension` and is covered by test.

Reply:
Thanks for the clarification!